ortools/linear_solver/xpress_interface.cc

```diff
@@ -1209,16 +1227,162 @@ void XpressInterface::SetLpAlgorithm(int value) {
   }
 }
 
+bool XpressInterface::readParameter(std::string const &name, std::string const &value)
```

Comment:
Adding functions to `XpressInterface` may not be necessary here. This class is used for external access through `MPSolver` functions. However, it won't be accessible generically.

I suggest a free function taking a reference to an `XPRSprob` object, preferably in the anonymous `namespace`.

Reply:
I moved this function into an anonymous namespace in commit 85db840e6292901e4355880f37d435261cb86642.
I also moved the other free functions into the anonymous namespace in commit 0f4395568122f1c494049fb6d18a3c0aac28690c.